tests/framework/installer/install_helper.go

```diff
@@ -24,37 +24,47 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/coreos/pkg/capnslog"
 	"github.com/rook/rook/tests/framework/contracts"
 	"github.com/rook/rook/tests/framework/transport"
 	"github.com/rook/rook/tests/framework/utils"
+	"strings"
 )
 
 const (
 	rookOperatorFileName   = "rook-operator.yaml"
 	rookClusterFileName    = "rook-cluster.yaml"
 	rookToolsFileName      = "rook-tools.yaml"
 	podSpecPath            = "src/github.com/rook/rook/demo/kubernetes"
+	podSpecPath1_5         = "src/github.com/rook/rook/demo/kubernetes/1.5"
```

Comment:
to support multiple versions it would be good factor out some properties to a new type or use an interface so the tests can be version agnostic